app/controllers/application_controller.rb

```diff
@@ -1,6 +1,13 @@
 class ApplicationController < ActionController::Base
+  include SessionsHelper
 
-  def hello
-    render html: 'hello'
+  private
+
+  def logged_in_user
+    unless logged_in?
```

Comment:
imgo, `return` always more readable than `if`, f.e.
`return if logged_in?`

Reply:
Did you mean I should replace this block with following rows:
`return store_location if logged_in?`
`return flash[:danger] = 'Please log in' if logged_in?`
`return redirect_to login_url if logged_in?`